redaxo/src/core/lib/util/i18n.php

```diff
@@ -63,6 +65,10 @@ public static function getLocale()
      */
     public static function addDirectory($dir)
     {
+        if (self::$cacheLoaded) {
```

Comment:
Wissen wir hier, dass der dir der hinzugefügt werden soll, auch einer ist der im cache enthalten war?

Reply:
Anstatt die methode aufrufen zu können und dann still und heimlich nichts zu tun fände ich es besser eine isCached() o.ä. methode zu haben, wo der caller selbst enscheiden kann wann addDirectory() aufgerufen wird (dann sieht man im aufrufenden code auch besser wann was passiert)

@gharlan meinung dazu?

Reply:
Hmm, ich habe mir gerade den PR angeschaut, und bin da nun drüber gestolpert (über die isCached()-Aufrufe von außen), und habe nun den Kommentar hier gefunden. Gestolpert bin ich, da wir das sonst nirgends so machen. Daher weiß ich nicht so recht, ob ich damit nun hier damit anfangen würde..
Ist der Cache nicht eine weitestgehend interne Angelegenheit der Klasse, und der Caller sollte sich darum nicht scheren müssen?

Reply:
Zum Zeitpunkt des Kommentars ging das noch nicht. Da wir mittlerweile die is-readable calls im calling code nicht mehr haben, kann man das caching jetzt auch in der klasse verstecken.